README.md

```diff
@@ -1,14 +1,15 @@
 # Service Fabric
 
-Service Fabric is a distributed systems platform for packaging, deploying, and managing stateless and stateful distributed applications and containers at large scale. Service Fabric runs on Windows and Linux, on any cloud, any datacenter, across geographic regions, or on your laptop.
+[Service Fabric is a distributed systems platform](https://github.com/GitTorre/service-fabric/tree/master/docs/architecture/README.md) for packaging, deploying, and managing stateless and stateful distributed applications and containers at large scale. Service Fabric runs on Windows and Linux, on any cloud, any datacenter, across geographic regions, or on your laptop.
 
 ## Getting started with Service Fabric
```

Comment:
This isn't specific to your PR but I think this section should be removed or placed further in the document. The GitHub page is NOT about people looking to USE service fabric. It's for people looking to contribute to the code base. People who come to this repo come in order to contribute, not to get started using it.

I mention this as context for below comment on the Youtube link